testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-kafka-ssl/secrets/create-certs.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
```

Comment:
have you used `shellcheck` for this file?

Reply:
Hi @kokosing , I did use shellcheck like you recommended https://www.shellcheck.net/ .

Reply:
![image](https://user-images.githubusercontent.com/5478260/111463866-705aae80-86f6-11eb-90e0-439f6ba69ed8.png)